packages/firebase_in_app_messaging/example/lib/main.dart

```diff
@@ -0,0 +1,122 @@
+import 'dart:async';
+
+import 'package:firebase_analytics/firebase_analytics.dart';
```

Comment:
I think we should simplify the example to focus on firebase_in_app_messaging and not include the analytics stuff. To reduce maintenance overhead, plugins should try to avoid unnecessary dependencies on other plugins, even in the examples, unless the other plugins are really necessary to understand how the plugin works. Sometimes it's not possible (e.g. firebase_database, firebase_auth) but here it doesn't seem like there's much interaction going on between analytics and fiam here. Let me know if you feel otherwise...

Reply:
Hmm, I think having the analytics trigger is a good idea. It's something users ask us about and the example shows doing that in Flutter is equally easy. So I think it's nice to keep since it doesn't add much complexity to the already straightforward example.

Reply:
OK, we can leave it there for now but we might remove it in a future release.